jcommune-plugins/poulpe-auth-plugin/src/main/java/org/jtalks/jcommune/plugin/auth/poulpe/service/PoulpeAuthService.java

```diff
@@ -296,6 +304,26 @@ protected ClientResource sendAuthRequest(String username, String passwordHash) {
         return clientResource;
     }
 
+    /**
+     * Send user activation request for specified username
+     * @param username username
+     * @return ClientResource result
+     */
+    protected ClientResource sendActivationRequest(String username){
+        String url = activationUrl + "?username=" + username;
+        ClientResource clientResource = createClientResource(url, false);
+        if (!StringUtils.isAnyBlank(login, password)) {
```

Comment:
Looks like it's repeated many times. Maybe something like `addCredentialsIfAny(clientResource)`?